### PR TITLE
I try to fix dead link from exo

### PR DESCRIPTION
**Pull request by Pbast**
I try to fix dead ling from http://extensions.xwiki.org/xwiki/bin/view/Extension/CKEditor Integration because "Automatically synchronize the extension informations" is checked

### CODE REVIEW

File: ui/pom.xml [hunk 1]
```diff
@@ -29,8 +29,8 @@
     <artifactId>application-ckeditor-pom</artifactId>
     <version>1.21-SNAPSHOT</version>
   </parent>
-  <artifactId>application-ckeditor-ui</artifactId>
-  <name>CKEditor Integration UI</name>
+  <artifactId>application-ckeditor</artifactId>
+  <name>CKEditor Integration</name>
```
Comment (vmassol) [inline ui/pom.xml]: Why did you change the artifact ID? That looks wrong ;)

File: ui/pom.xml [hunk 2]
```diff
@@ -89,4 +89,4 @@
       <type>xar</type>
     </dependency>
   </dependencies>
-</project>
\ No newline at end of file
+</project>
```
Comment (vmassol) [inline ui/pom.xml]: change not needed here I think.

Comment (Pbast) [reply ui/pom.xml]: yes because I used github to edit pom.xml

**Comment by vmassol** [comment]
Instead you need something like https://github.com/Pbast/application-ckeditor/blob/79369fe19fc91665794c7255f42e516fb9db3d50/pom.xml#L46

File: ui/pom.xml [hunk 3]
```diff
@@ -30,7 +30,7 @@
     <version>1.21-SNAPSHOT</version>
   </parent>
   <artifactId>application-ckeditor-ui</artifactId>
-  <name>CKEditor Integration UI</name>
+  <name>CKEditor Integration</name>
```
[the remark this answers is not in the document]
Comment (Pbast) [reply ui/pom.xml]: Ok I give up I don't know where/how to fix it (I don't understand what I'm doing with jenkins). Now I wonder if href must fixed here on pom because <url>https://github.com/xwiki-contrib/application-ckeditor/tree/master</url> is correct but wrong on exo page

[the remark this answers is not in the document]
Comment (vmassol) [reply ui/pom.xml]: The XWiki Repository Application gets the URL from the Maven `<scm><url>` value (see https://github.com/Pbast/application-ckeditor/blob/79369fe19fc91665794c7255f42e516fb9db3d50/pom.xml#L46).

**Comment by Pbast** [comment]
url in pom .xml is correct, the issue is probably in the script/engine (?) that generate the page http://extensions.xwiki.org/xwiki/bin/view/Extension/CKEditor Integration

**Comment by Pbast** [comment]
Yes it was for that I changed artifact id first (by guessing) but I hate doing what I don't understand at all. Then i closed  this PR.
